imagersite/imager_images/tests.py

```diff
@@ -85,6 +89,25 @@ def test_user_can_see_its_album_description(self):
         one_user = User.objects.first()
         self.assertEqual(one_user.albums.first().description, one_user.albums.first().description)
 
+    def test_user_can_get_to_album_creation(self):
+        """Test user can see album form."""
+        user = User.objects.first()
+        self.client.force_login(user)
+        response = self.client.get('/images/albums/add/')
+        self.assertEqual(response.status_code, 200)
+
+    def test_user_cant_get_to_album_creation_if_not_logged_in(self):
+        """Test user can see album form."""
+        response = self.client.get('/images/albums/add/')
+        self.assertEqual(response.status_code, 302)
+
+    def test_create_album(self):
+        """Test for album creation."""
+        user = User.objects.first()
+        self.client.force_login(user)
+        response = self.client.post('/images/albums/add/', {'cover': '', 'title': 'title', 'description': 'description', 'published': 'PRIVATE'})
```

Comment:
Try and avoid having lines that are exceedingly long like this as it makes your code difficult to read. You can easily break this into multiple lines to make it more readable.
```python
response = self.client.post('/images/albums/add/', 
    {
        'cover': '',
        'title': 'title',
        'description': 'description',
        'published': 'PRIVATE'
})
```